integration-tests/ci-visibility.spec.js

```diff
@@ -0,0 +1,113 @@
+'use strict'
+
+const { fork } = require('child_process')
```

Comment:
I can't use `spawnProc` as is, as it returns a promise and I need to handle `stdout` and `stderr` right away. A vanilla `fork` works for my use case though so this should be fine